src/pages/contributing/components.mdx

```diff
@@ -87,3 +88,125 @@ For guidance, see our Documentation requirements section above or reach out to u
 
 The component for **Carbon for IBM.com** must be built in one of our supported frameworks - React or Web Components.  We'd like to prioritize working on Web Components since it has become our strategic direction.  If you need help learning how to build it in Web Components, reach out to the engineers in our slack channel for training <a href="https://ibm-studios.slack.com/archives/C2PLX8GQ6" target="_blank">#carbon-for-ibm-dotcom</a>.  Please read the [Developer guide](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/docs/developing.md) and the [Submissions guidelines](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/docs/submission-guidelines.md) for more details.
 
+## How to write component guidance
```

Comment:
I think it should be writing guidelines for the component as well. The current title is "How to write component guidance". Are you suggesting to have the title as "How to write a component guideline", James?

Reply:
"Structure" doesn't make much sense to me here personally. Maybe something more like...

"Component guideline publishing"
"How to publish component guidelines"
"Writing component guidelines"

Just some ideas...

Reply:
Updated the tiel to "How to publish component usage documentation", since we list the document as "usage documentation" in the Component contribution section.